config/_ytt_lib/github.com/cloudfoundry/cf-k8s-networking/config/cfroutesync/istio-gateway.yaml

```diff
@@ -1,17 +0,0 @@
-#@ load("@ytt:data", "data")
----
```

Comment:
Why are we switching the apps over to using the `cf-system` Gateway?

I think we'll also need to update these docs: https://github.com/cloudfoundry/cf-k8s-networking/blob/master/doc/managing-certificates-for-application-ingress.md

Reply:
Having multiple gateways doesn't buy us anything. If two gateways have conflicting configuration Istio will silently accept them and then generate invalid configuration to ship to the ingress gateways. Better in my opinion to have a single gateway as the source of truth where we can see if we've accidentally tried to make two `servers` with the same wildcard host.

Also, app domains aren't necessarily tied to a given space or org, so in the ideal future where spaces are mapped to namespaces it would leave us with a complicated tangle of gateways.